vm/qemu/qemu.go

```diff
@@ -202,8 +204,14 @@ var archConfigs = map[string]*archConfig{
 		RngDev:   "virtio-rng-pci",
 	},
 	"darwin/amd64": {
-		Qemu:      "qemu-system-x86_64",
-		QemuArgs:  "-enable-kvm -machine q35 -cpu host,migratable=off",
+		Qemu: "qemu-system-x86_64",
+		QemuArgs: strings.Join([]string{
+			"-accel hvf -machine q35 ",
+			"-cpu Penryn,vendor=GenuineIntel,+invtsc,vmware-cpuid-freq=on,",
+			"+pcid,+ssse3,+sse4.2,+popcnt,+avx,+aes,+xsave,+xsaveopt,check ",
+			"-drive if=pflash,format=raw,readonly=on,file=/usr/local/share/OVMF/OVMF_CODE.fd ",
```

Comment:
Maybe OVMF base path is also worth being a separate parameter. What do you think?

Reply:
ack!